cmd/consensusGenome/consensusGenome.go

```diff
@@ -25,16 +27,53 @@ var Technologies = map[string]string{
 var technologyOptionsString string
 
 var WetlabProtocols = map[string]string{
```

Comment:
More as follow up... this cli hard code does not seem sustainable. Should we to create API endpoints for this info?

Reply:
I had the same thought. It was hard-coded in the web client so I hard-coded it here. On one hand it is a huge pain but on the other these are listed in the help text and I am not sure we should require network connectivity for that. The hardcoding may be the lesser of two evils.